### DISCUSSION TITLE
AAD Approval Required for Seq in ACI with AAD integration

### DISCUSSION THREAD

**Discussion by amitjaura** [datalust/seq-tickets#2029] [Help]
Hi,

Our Seq instance is hosted in ACI and linked to AAD. Recently we have seen new users trying to log into Seq is presented with below option. It used to be simple onboarding. Has something changed recently?

<img width="217" alt="image" src="https://github.com/datalust/seq-tickets/assets/25460755/17437f4b-7636-43bd-a857-adc09eed7a1b">

**Comment by liammclennan** [comment]
Hi @amitjaura ,

This looks like an AAD configuration issue. Have you tried your Azure support?
